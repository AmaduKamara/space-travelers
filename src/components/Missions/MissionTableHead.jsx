import React from 'react';

const MissionTableHead = () => (
  <>
    <thead className="bg-white border-b">
      <tr>
        <th scope="col" className="text-sm font-bold text-gray-900 px-6 py-4 text-left border">
          Mission
        </th>
        <th scope="col" className="text-sm font-bold text-gray-900 px-6 py-4 text-left border">
          Description
        </th>
        <th scope="col" className="text-sm font-bold text-gray-900 px-6 py-4 text-left border">
          Status
        </th>
        <th scope="col" className="text-sm font-bold text-gray-900 px-6 py-4 text-left">
          {' '}
        </th>
      </tr>
    </thead>
  </>
);

export default MissionTableHead;
