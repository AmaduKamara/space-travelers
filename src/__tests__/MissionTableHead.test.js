import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import MissionTableHead from '../components/Missions/MissionTableHead';

const renderer = require('react-test-renderer');

it('matches snapshot of MissonTableHead component', () => {
  const tree = renderer.create(<MissionTableHead />).toJSON();
  expect(tree).toMatchSnapshot();
});
