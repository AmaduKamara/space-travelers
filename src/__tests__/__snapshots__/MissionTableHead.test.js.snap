// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot of MissonTableHead component 1`] = `
<thead
  className="bg-white border-b"
>
  <tr>
    <th
      className="text-sm font-bold text-gray-900 px-6 py-4 text-left border"
      scope="col"
    >
      Mission
    </th>
    <th
      className="text-sm font-bold text-gray-900 px-6 py-4 text-left border"
      scope="col"
    >
      Description
    </th>
    <th
      className="text-sm font-bold text-gray-900 px-6 py-4 text-left border"
      scope="col"
    >
      Status
    </th>
    <th
      className="text-sm font-bold text-gray-900 px-6 py-4 text-left"
      scope="col"
    >
       
    </th>
  </tr>
</thead>
`;
